source/core/NumericString.cpp

```diff
@@ -2090,7 +2090,7 @@ struct ByteArrayToStringParamBlock : InstructionCore
 static bool CheckUnsupportedEncodingError(ErrorCluster* errorCluster, UInt16 stringEncoding, TypedArrayCoreRef outputArray) {
     if (stringEncoding != 0) {
         if (errorCluster != nullptr) {
-            errorCluster->SetErrorAndAppendCallChain(true, 1, "Encoding type not supported");
+            errorCluster->SetErrorAndAppendCallChain(true, 1396, "Encoding type not supported");
```

Comment:
@sanmut described updating the SetErrorAndAppendCallChain() call to use the VI name, ie

```cpp
errorCluster->SetErrorAndAppendCallChain(true, 1396, "Byte Array To String or whatever");
```

But it looks like all the calls to SetErrorAndAppendCallChain in this file are incorrectly giving a "supplementary" error instead of the VI Name.

I believe the callsites in this file should be updated similar to how the [Format function reports errors](https://github.com/ni/VireoSDK/blob/master/source/core/NumericString.cpp#L276).

ie

```cpp
errorCluster->SetErrorAndAppendCallChain(true, 1396, "Byte Array To String or whatever");
errPtr->AddAppendixPreamble();
errPtr->source->AppendCStr("Encoding type not supported");
errPtr->AddAppendixPostamble();
```

Reply:
@rajsite. the fake PR to pull in my changes resulted in a single failing test: https://ni.visualstudio.com/DevCentral/_git/ASW/pullrequest/80900
I'll fix it while making the rest of these changes (updating the error string will result in needing to update a few C# tests).

Reply:
@rajsite, Good point, Agreed. I think we need to cleanup the error message handling throughout that has not been prioritized yet. Now, there are different levels of badness in different places in the code.

Reply:
@rajsite, @sanmut, and @cglzaguilar, I've made some changes per your requests.  I had to add a parameter to one function so it could correctly print out the proper error message (String To Byte vs Byte To String).  The AzDo build just started but should have results in the next hour and a half here: https://ni.visualstudio.com/DevCentral/_git/ASW/pullrequest/80900.